pyston/resource.py

```diff
@@ -710,7 +710,7 @@ def _get_form(self, fields=None, inst=None, data=None, files=None, initial={}):
             kwargs['files'] = files
 
         form_class = self._generate_form_class(inst, exclude)
-        return form_class(initial=initial, **kwargs)
+        return form_class(initial=initial or {}, **kwargs)
```

Comment:
initial může být prázdný dict nebo list nebo false a stále to bude fungovat, to není správně

má být pod hlavičkou metody řádek

initial = {} if initial is None else initial